### PR TITLE
fix: github link + gif example

### PR DESCRIPTION
**Pull request by desaintvincent**

### CODE REVIEW

**Review by NilinC — APPROVED** [review]
good

File: package.json [hunk 1]
```diff
@@ -1,8 +1,14 @@
 {
   "name": "angular-omnibox",
-  "version": "1.2.0",
+  "version": "1.2.1",
   "description": "omnibox for angular",
   "main": "src/js/module.js",
+  "repository": {
```
Comment (wixyvir) [inline package.json:6]: add https://docs.npmjs.com/files/package.json#homepage

**Review by wixyvir — CHANGES_REQUESTED** [review]
*Every Pull Request MUST be posted in #code-review so everyone can say what they have to say and not only people that you like*

**Review by bvibrac — APPROVED** [review]
Good! Proper Readme :+1:

**Comment by kevinbalicot** [comment]
:+1:

**Review by wixyvir — APPROVED** [review]

**Review by kevinbalicot — APPROVED** [review]
:+1:

**Review by jubianchi — APPROVED** [review]

**Pull request merged**